Controller/AutoCompleteController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Opifer\CmsBundle\Controller;
+
+use FOS\RestBundle\Controller\Annotations\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+class AutoCompleteController extends Controller
+{
+    /**
+     * @Route("/autocomplete/{alias}", name="opifer.cms.autocomplete", options={"expose"=true})
```

Comment:
Could you add the route to the `yml` routing files and use `_` instead of `.` in the route name?